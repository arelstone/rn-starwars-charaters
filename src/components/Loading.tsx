import React, { FC } from 'react';
import { View, Text, StyleSheet } from 'react-native';

interface LoadingProps { }

const Loading: FC<LoadingProps> = () => {
    return <View style={styles.container}>
        <Text style={styles.text}>Loading</Text>
    </View>;
};

export default Loading;

const styles = StyleSheet.create({
    container: {
        flex: 1,
        justifyContent: 'center',
        alignItems: 'center',
    },
    text: {
        fontSize: 26,
    },
});
