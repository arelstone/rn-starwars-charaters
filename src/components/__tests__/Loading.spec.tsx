import React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';
import Loading from '../Loading';

let wrapper: ShallowWrapper;

describe('<Loading />', () => {
    beforeAll(() => {
        wrapper = shallow(<Loading />);
    });

    it('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
