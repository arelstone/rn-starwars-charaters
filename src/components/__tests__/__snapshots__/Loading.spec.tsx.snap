// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 26,
      }
    }
  >
    Loading
  </Text>
</View>
`;
